---
title: "UnboundLocalError when `mesolve` is used with callable `e_ops`"
html_url: "https://github.com/qutip/qutip/issues/2118"
user: "balopat"
repo: "qutip/qutip"
---

